packages/accordion/src/Accordion.js

```diff
@@ -0,0 +1,175 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { cx, css } from "emotion";
+import { ThemeContext } from "@hig/theme-context";
+import { ControlBehavior } from "@hig/behaviors";
+
+import {
+  indicators,
+  indicatorPositions,
+  AVAILABLE_INDICATORS,
+  AVAILABLE_INDICATOR_POSITIONS
+} from "./constants";
+import stylesheet from "./stylesheet";
+import HeaderPresenter from "./presenters/HeaderPresenter";
+import ContentPresenter from "./presenters/ContentPresenter";
+
+export default class Accordion extends Component {
+  static propTypes = {
+    /** Text label for the accordion header */
+    label: PropTypes.string.isRequired,
+    /** Width of the accordion */
+    width: PropTypes.string,
```

Comment:
should we have a `width` prop? shouldn't the `accordion` take the width of its container?

Reply:
Hi @wmui51 , yes the component will take the width of its container by default. (the samples in acceptance actually are using this default behavior). 
I just referred https://hig.autodesk.com/web/components/accordion which marked "title", "indicator" and "**width**" in Anatomy section. So i guess it's designed to have a `width` property. 
Let me know if we should remove this prop and just keep using its parent's width.

Reply:
I think we should remove the `width` prop and let it take the size of the container and let users control that. If there is a use case for this prop it would be easier to add later than to remove a prop.

Reply:
ok, i have removed the `width` prop